data-raw/england-ltla-aande.R

```diff
@@ -0,0 +1,69 @@
+library(tidyverse)
+library(geographr)
+library(healthyr)
+library(compositr)
+library(sf)
+
+ltla <-
+  boundaries_ltla21 |>
+  st_drop_geometry() |>
+  filter(str_detect(ltla21_code, "^E"))
+
+trust_names <-
+  points_nhs_trusts22 |>
+  st_drop_geometry() |>
+  select(starts_with("nhs_"))
+
+# ---- Attendances over 4 hours ----
+attendances_4hours_trust <- england_trust_accidents_emergency |>
+  mutate(date = parse_date_time(date, orders = "B Y")) |>
+  filter(date >= ymd("2022-05-01"), date <= ymd("2023-04-01")) |>
+  subset(select = c(nhs_trust22_code, attendances_over_4hours, pct_attendance_over_4hours)) |>
+  mutate(pct_attendance_over_4hours = case_when(
+    attendances_over_4hours == 0 ~ 0,
+    TRUE ~ pct_attendance_over_4hours
+  ))|> # Replace NA in pct column with 0 when its value is actually 0
+  group_by(nhs_trust22_code) |>
+  summarize(number_sum = sum(attendances_over_4hours), percent_mean = mean(pct_attendance_over_4hours)) |>
```

Comment:
In you calls to `sum()` and `mean()`, set the arg `na.rm = TRUE`. This will mean that `NA` values are ignored in the calculation, which is what we want.

Reply:
Do the same to future calls to `sum()` and `mean()` below.